corehq/apps/app_manager/suite_xml/sections/details.py

```diff
@@ -589,4 +590,4 @@ def _case_tile_template_string(self):
         with open(os.path.join(
                 os.path.dirname(os.path.dirname(__file__)), "case_tile_templates", "tdh.txt"
         )) as f:
```

Comment:
I think this should say `), encoding="utf-8") as f:`. Otherwise it uses [whatever `locale.getpreferredencoding()` returns](https://docs.python.org/2/library/io.html#io.open), which is probably ASCII.

This type of change is probably necessary in most other places that are not using binary mode (`wb`, `rb`, etc.).

Reply:
Ah, I didn't realize ```locale.getpreferredencoding()``` was the default.  I'll make those updates.

Reply:
https://github.com/dimagi/commcare-hq/pull/20611